web/src/classic/components/molecules/Visualizer/Engine/Cesium/Tileset/index.tsx

```diff
@@ -232,13 +232,23 @@ const Tileset: FC<PrimitiveProps<Property, any, SceneProperty>> = memo(function
       : null;
   }, [isVisible, sourceType, tileset, meta, googleMapResource]);
 
+  const handleOnReady = useCallback(
+    (t: Cesium3DTilesetType) => {
+      if (_debugFlight) {
+        viewer?.flyTo(t);
+      }
+      setIsTilesetReady(true);
+    },
+    [viewer],
+  );
```

Comment:
Just to avoid any unnecessary re-renders because of unnecessary state updates.

```suggestion
  const handleOnReady = useCallback(
    (t: Cesium3DTilesetType) => {
      if (isTilesetReady) return;
      if (_debugFlight) {
        viewer?.flyTo(t);
      }
      setIsTilesetReady(true);
    },
    [viewer],
  );
```

Reply:
The event happens only once, also it's just boolean, so it doesn't cause unnecessary render.